src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
+
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + at + ")";
+    }
+
+    @Override
+    public String doneToString() {
```

Comment:
Maybe you could change this function's name? It could help indicate what the function does.